### ISSUE TITLE
Oredictionificator not accepting block oredict entries

### ISSUE THREAD

**Issue by Sil3ntStorm** [aidancbrady/Mekanism#3124]
It seems that the Oredictionificator does not accept oredict entries that start with block. I tried to get it to unify RP2 Marble and Chisel Marble, both of which are blockMarble in oredict. However it will never do anything if clicking the button on the right or hitting enter. It works fine with entries starting with ore or ingot, but other than that it doesn't seem to do anything.
Also it's impossible to have more than 3 entries just as described in #2676 with every new one editing the last instead.
Using Mekanism 8.1.8

**Comment by laz2727** [comment]
It only works with ores, ingots, nuggets or dusts. That's intentional.

**Comment by aidancbrady** [comment]
Indeed. If I were to allow blocks, it would be possible to create any type of stained glass from a block of generic glass just by passing it through the machine- this is just one issue of many.

**Comment by JaSpr** [comment]
This prevents IC2 Uranium from being converted to "Immersive Engineering" or "Extreme Reactors" uranium in any form.  Once Yellorium (ER) is merged into blocks, it becomes IC2 blocks, which breaks up into pure uranium (with lots of radiation, so difficult to work with).  That uranium, since it is not an "ore", can't be converted to IE Uranium or ER Yellorium.  And the IC2 Block (obviously) can't be converted either, so you're left with blocks of uranium that you can't do anything with. (Trying to merge all my uranium into IE Uranium Ingots so I can put it into a Quantum Storage unit to wait until down the road when I'm ready to start exploring nuclear energy.)
